numpy/testing/__init__.py

```diff
@@ -14,6 +14,7 @@
 from ._private.nosetester import (
     run_module_suite, NoseTester as Tester
     )
+from . import overrides
```

Comment:
Does this end up in `__all__`?  I am surprised the public API test does not fail?